src/command/verify/verify.c

```diff
@@ -1632,6 +1632,63 @@ verifyProcess(unsigned int *errorTotal)
     FUNCTION_LOG_RETURN(STRING, result);
 }
 
+/***********************************************************************************************************************************
+Format the text result
+***********************************************************************************************************************************/
+static String *
+verifyOutputText(const String *verifyresult, const unsigned int *errorTotal)
+{
+    FUNCTION_LOG_BEGIN(logLevelDebug);
+        FUNCTION_TEST_PARAM(STRING, verifyresult);                  // Result string from verifyProcess()
+    FUNCTION_LOG_END();
+
+    String *result = NULL;
+
+    MEM_CONTEXT_TEMP_BEGIN()
+    {
+        String *resultStr = strNew();
+        String *status = strNew();
+
+        if (*errorTotal == 0)
+            strCatFmt(status,"%s%s",  strZ(strNewZ("All archives and backups passed verification.")), LF_Z);
+        else
+        {
+            strCatFmt(
+                status,
+                "%s%s",
+                strZ(strNewZ("One or more archive or backup contains errors. Invoke verify with verbose option for more details.")),
```

Comment:
Hey,
Shouldn't this output `Invoke verify with verbose option for more details.` be removed if the verbose option is already provided? It sounds a bit weird to request it while the user already provided it :confused:

Reply:
Also wondering, since this is "one or more", perhaps `archive(s) or backup(s)` would make sense? :thinking:

Reply:
addressed in 1ab4e8eb20873848a78df60be92e2356098786ec

Reply:
Sounds better, thanks :+1: